http/utility.hpp

```diff
@@ -610,6 +612,17 @@ inline std::string getDateTime(const std::time_t& time)
     return redfishDateTime;
 }
 
+inline std::string getDateTimeUintMs(uint64_t millisSecondsSinceEpoch)
+{
+
+    boost::posix_time::milliseconds timeSinceEpoch =
+        boost::posix_time::milliseconds(millisSecondsSinceEpoch);
+    boost::posix_time::ptime epoch(boost::gregorian::date(1970, 1, 1));
+    boost::posix_time::ptime time = epoch + timeSinceEpoch;
```

Comment:
Did you get this from anywhere?

Reply:
I got this from upstream's http/utility.hpp mostly. I thought if i did it this way it would make the jump to upstream for future patches smaller.